modules/morphology/src/main/resources/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.1.json

```diff
@@ -0,0 +1,81 @@
+{
+  "@context": [
+    "{{base}}/contexts/neurosciencegraph/core/schema/v0.1.0",
+    {
+      "this": "{{base}}/schemas/neurosciencegraph/morphology/fixationstainingmounting/v0.1.0/shapes/"
+    }
+  ],
+  "@type": "nxv:Schema",
+  "imports": [
+    "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.2"
+  ],
+  "shapes": [
+    {
+      "@id": "this:FixationStainingMountingShape",
+      "@type": "sh:NodeShape",
+      "label": "Fixing, Staining and Mounting activities description",
+      "targetClass": "nsg:FixationStainingMounting",
+      "nodeKind": "sh:BlankNodeOrIRI",
+      "and": [
+        {
+          "node": "{{base}}/schemas/neurosciencegraph/commons/activity/v0.1.2/shapes/ActivityShape"
+        },
+        {
+          "property": [
+            {
+              "path": "nsg:fixationMethod",
+              "name": "Fixation method",
+              "description": "Method used for the fixation of the slice after electrophysiology such as PFA.",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:stain",
+              "name": "Stain",
+              "description": "Stain used for the detection of the cell labeling compound such as the chromogen DAB.",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "nsg:mountingMedia",
+              "name": "Mounting Media",
+              "description": "Mounting media is the medium that the sample is in while it is being imaged on the microscope such as Xylene or Mowiol.",
+              "datatype": "xsd:string",
+              "maxCount": 1
+            },
+            {
+              "path": "prov:used",
+              "description": "The patched slice used",
+              "xone": [
+                {
+                  "class": "nsg:PatchedSlice"
+                },
+                {
+                  "class": "nsg:Collection"
+                }
+              ],
+              "minCount": 1,
+              "maxCount": 1,
+              "seeAlso": "{{base}}/schemas/neurosciencegraph/experiment/patchedslice/v0.1.0/shapes/PatchedSliceShape"
+            },
+            {
+              "path": "prov:generated",
+              "description": "The fixed and stained slice used",
+              "xone": [
```

Comment:
Looks like a cardinality difference. I don't think this should be handled by xone.